test/e2e/kubectl.go

```diff
@@ -202,6 +202,29 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl run", func() {
+		It("should run a particular image on the cluster", func() {
+			nsFlag := fmt.Sprintf("--namespace=%v", ns)
+			image := "nginx"
+			rcName := "e2e-test-nginx-rc"
+
+			By("running the image " + image)
+			runKubectl("run", rcName, "--image="+image, nsFlag)
+			By("verifying the rc " + rcName + " was created")
+			output := runKubectl("get", "rc", nsFlag)
+			if !strings.Contains(output, rcName) {
+				Failf("Failed creating rc " + rcName)
+			}
+			By("verifying the pod controlled by rc " + rcName + " was created")
+			output = runKubectl("get", "pods", nsFlag)
```

Comment:
Instead of getting all pods, you can run
```
get pods -l run=nginx
```
The `run=imagename` label is applied to all pods created by the run command.